src/ui/directives/query/queryCtrl.js

```diff
@@ -83,6 +83,10 @@ angular.module('app').controller('queryCtrl', [
       modalService.openQueryResultsExport($scope.currentCollection, $scope.currentQuery);
     };
 
+    $scope.collapseAll = function() {
+      $rootScope.$broadcast('collapse');
```

Comment:
Could we change this to just $scope.$broadcast? I pulled down your code and it seems to work fine that way. This way we lessen the chance of something else in the app re-using this same event name and causing some weird results.